.github/workflows/automerge.yml

```diff
@@ -36,11 +36,9 @@ jobs:
     steps:
       - name: Automerging
         uses: pascalgn/automerge-action@v0.7.5
-        if: github.actor == 'asyncapi-bot' || github.actor == 'dependabot[bot]' || github.actor == 'dependabot-preview[bot]'
         env:
           GITHUB_TOKEN: "${{ secrets.GH_TOKEN }}"
           GITHUB_LOGIN: asyncapi-bot
-          MERGE_LABELS: ""
```

Comment:
why removed? docs say `When an empty string ("") is given, all pull requests will be merged.`
shouldn't you do `MERGE_LABELS: automerge`?

anyway I don't like the change as now we have issue with dependabot again and also the action I just finished yesteday is not supporting this approach. Why is this change needed anyway? why can't we merge automatically once `Update dependencies` is successfully completed?

Reply:
We can't because it will also merge breaking changes automatically. For instance, if it receives an update to a major version it will get updated on the package.json but the API probably changes and therefore it will fail. 🤔 but now that I think about it, tests would fail 🤔

Reply:
exactly, they should fail, and if didn't, not a breaking change, 
parser 2.0 might not require to release generator 2.0 (in theory, it will anyway fail because of tests)